README.md

```diff
@@ -144,3 +144,42 @@ The following is the request flow (seen in blue in the architecture diagram abov
     ```
 
 1. You can see the pods with `kubectl get pods.`
+
+## Uninstalling the component
+1. Remove the demo application
+    ```
+    kubectl delete -f test/app/service.yml
+    ```
+2. If you previously set every service to respond to the `Prefer: respond-async` header, remember to remove this setting by updating the ingress.class to null or by updating the ingress.class to the ingress.class you would like to use, for example `kourier`.
+    ```
+    kubectl patch configmap/config-network \
+    -n knative-serving --type merge \
+    -p '{"data":{"ingress.class":null}}'
+    ```
+
+    ```
+    kubectl patch configmap/config-network \
+    -n knative-serving \
+    --type merge \
+    -p '{"data":{"ingress.class":"kourier.ingress.networking.knative.dev"}}'
+    ```
+3. Remove the producer component
```

Comment:
For numbered lists, you can just use `1.` for each item, and it will automatically be picked up as the proper numbering. This helps if these steps have to change in the future - we won't need to renumber them :)